### DISCUSSION TITLE
Change icon of dropdownIndicator?

### DISCUSSION THREAD

**Discussion by QuangLoc0311** [csandman/chakra-react-select#181] [Q&A] [answered]
How can I change icon of dropdownIndicator like this?

![Screen Shot 2022-09-14 at 09 24 03](https://user-images.githubusercontent.com/32577018/190044486-b8455f6b-9be6-4c11-b6f2-543b5ca17efd.png)

**Comment by csandman** [comment, marked as answer]
Check out the [Customizing Components](https://github.com/csandman/chakra-react-select#customizing-components) section of the docs.  The example listed there should be exactly what you need: https://codesandbox.io/s/chakra-react-select-custom-icons-xf7scd?file=/example.js

For simplicities sake, here's a specific example:

```jsx
import { Icon } from "@chakra-ui/react";
import { Select, chakraComponents } from "chakra-react-select";
import { AiFillCaretDown } from "react-icons/ai";

const customComponents = {
  DropdownIndicator: (props) => (
    <chakraComponents.DropdownIndicator {...props}>
      <Icon as={AiFillCaretDown} h={4} w={4} />
    </chakraComponents.DropdownIndicator>
  )
};

const Example = () => (
  <Select
    useBasicStyles
    components={customComponents}
  />
);
```
